src/services/TokenService.ts

```diff
@@ -106,7 +106,7 @@ class TokenService {
     if (data === undefined) {
       throw new Error("Unknown token " + coinSymbol)
     }
```

Comment:
I still don’t understand why we have to make a copy. These objects are supposed to be “immutable”